### DISCUSSION TITLE
latest version of Smite lib not working

### DISCUSSION THREAD

**Discussion by FODZERD** [thatskymod/Sky-CotL-Scripts#53] [Q&A]
in android device when i open game its will crash immediately, the most people have a same problem, pls fix it

**Comment by thatskymod** [comment]
Live crash fixed in [libsmite-1.0.7d.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libsmite-1.0.7d.so)
